server/src/main/java/io/crate/metadata/sys/SysShardsTableInfo.java

```diff
@@ -197,6 +199,7 @@ public static SystemTable<ShardRowContext> create(Roles roles) {
                 .add("periodic_count", LONG, ShardRowContext::flushPeriodicCount)
                 .add("total_time_ns", LONG, ShardRowContext::flushTotalTimeNs)
             .endObject()
+            .add(Columns.LAST_WRITE_BEFORE.name(), DataTypes.TIMESTAMPZ, r -> r.indexShard().getLastWriteNanos())
```

Comment:
The internal representation for timestamp is milliseconds

Reply:
So this should be `LONG` rather than `TIMESTAMPZ`?

Reply:
No, the nanos need to be converted to millies so that clients get a sensible timestamp as result.